src/__Private/Visitor/TypeInfo.hack

```diff
@@ -91,6 +91,16 @@ final class TypeInfo extends ASTVisitor {
                     throw new \Slack\GraphQL\UserFacingError("Unrecognized type: %s", $node->getType());
             }
             $this->type_stack->push($type);
+        } else if (
+            $node is Parser\FragmentSpread\InlineFragmentSpread ||
+            $node is Parser\Fragment\Fragment
+        ) {
+            $schema = $this->schema;
```

Comment:
We should pass down an instance of the schema, rather than the schema type itself, but I want to make that change separately.